esmvalcore/preprocessor/_regrid.py

```diff
@@ -292,54 +303,48 @@ def _spec_to_latlonvals(
 
     Parameters
     ----------
-    start_latitude : float
+    start_latitude:
         Latitude value of the first grid cell center (start point). The grid
         includes this value.
-    end_latitude : float
+    end_latitude:
         Latitude value of the last grid cell center (end point). The grid
         includes this value only if it falls on a grid point. Otherwise, it
         cuts off at the previous value.
-    step_latitude : float
+    step_latitude:
         Latitude distance between the centers of two neighbouring cells.
-    start_longitude : float
+    start_longitude:
         Latitude value of the first grid cell center (start point). The grid
         includes this value.
-    end_longitude : float
+    end_longitude:
         Longitude value of the last grid cell center (end point). The grid
         includes this value only if it falls on a grid point. Otherwise, it
         cuts off at the previous value.
-    step_longitude : float
+    step_longitude:
         Longitude distance between the centers of two neighbouring cells.
 
     Returns
     -------
-    xvals : np.array
+    xvals: np.array
         List of longitudes
-    yvals : np.array
+    yvals: np.array
         List of latitudes
```

Comment:
This isn't valid syntax, maybe something like?
```
tuple[np.ndarray, np.ndarray]
    List of longitudes, list of latitudes
```

Reply:
Changed in https://github.com/ESMValGroup/ESMValCore/pull/2772/commits/69e05029bca9d467e4f6b401ef8e506e652df23e